packages/big-design/src/components/Datepicker/Header.tsx

```diff
@@ -0,0 +1,48 @@
+import { ChevronLeftIcon, ChevronRightIcon } from '@bigcommerce/big-design-icons';
+import React from 'react';
+
+import { Flex } from '..';
+import { Button } from '../Button';
+import { Text } from '../Typography';
+
+export interface HeaderProps {
```

Comment:
I also don't see a need to export this too.

Reply:
Fixed that.